docs/FAQ/FAQ-Wallet.md

```diff
@@ -1,133 +1,77 @@
+## Can I use a hardware wallet with BTCPay Server?
 
-This happens because when you use a new derivation scheme with your store, BTCPay Server doesn't know the past transactions of your wallet.
+The internal wallet has a [built in hardware wallet integration](/Vault.md). You can use a supported hardware wallet with the [BTCPay wallet](Wallet.md). 
 
-To solve this issue, you need to use the **Rescan feature**. The Rescan relies on Bitcoin Core 0.17.0's [scantxoutset](https://bitcoincore.org/en/doc/0.17.0/rpc/blockchain/scantxoutset/) to scan the current state of the blockchain (called UTXO Set) for coins belonging to your derivation scheme.
-Once the scan is complete, BTCPay Server will show the correct balance, along with some of past transactions of your wallet.
+This  means that you're using a hardware wallet without leaking information to third-party apps or servers, since the wallet relays on the full node in your BTCPay.
+ 
+## Do I have to use BTCPay Server wallet?
 
-## Recommended external wallets?
+By default BTCPay Server only requires an extended public key. To receive payments to your BTCPay store, you need to provide an extended public key (xpub) which you can generate in an external wallet. You do not have to use a built in wallet at all, you can manage funds in your [existing wallet](/WalletSetup/#use-an-existing-wallet).
 
-We highly recommend you to use an internal wallet in case you need to double-check the transaction. If you wish to use an external wallet, here are our recomendations:
-
-* Electrum - desktop wallet
-* Ledger Nano S - hardware wallet (other hardware wallets should be added at some point)
-* Zap - Lightning Network LND wallet
-* Spark - Lightning Network c-lightning wallet
-* Sentinel - Watch-only on-chain wallet for Android
-* ArcBit / bitWallet - Watch-only on-chain wallet for Android
-
-## How to connect my wallet to BTCPay Server?
-
-The process of connecting a wallet to your BTCPay server is the same for both third-party or self-hosted deployments. Your private keys are never uploaded nor required by the BTCPay Server. You’re only uploading the xpubkey.
-
-There are two ways to connect your BTCPay and your wallet:
-
-1. Automatic with Ledger Nano S hardware wallet
-2. Manual with any wallet supporting xpubkey (Electrum highly recommended)
-
-Watch the video below or read [this article](https://bitcoinshirt.co/how-to-create-store-accept-bitcoin/8/#Connecting-BTCPay-with-your-wallet) for step by step instructions.
-
-[![ConnectBTCPayWallet](https://img.youtube.com/vi/xX6LyQej0NQ/mqdefault.jpg)](https://www.youtube.com/watch?v=xX6LyQej0NQ "BTCPay Server - Connecting Wallet")
+However, it's recommended to use a built in wallet for funds management. The built in wallet improves your privacy by default, but also solves user-experience issues like [gap-limit](#missing-payments-in-my-software-or-hardware-wallet).
```

Comment:
- There might be missing a warning here that the internal wallet can be a hot wallet if setup that way and that there can be loss of funds for XYYZ reasons.

- On that note, the word "recommended" might be too much. Why would we recommend a hot wallet instead of a HWW xpub setup ?

- The initial question being do I HAVE TO use the internal wallet, maybe there could be a list stating the usages that absolutely need the internal hot wallet to work (ie, payjoin ?) and that other usecases can be done either way ?

Reply:
No this has nothing to do with a hot wallet, nor is hot wallet recommended.

Reply:
@Zaxounette you probably need to understand wallet a bit more, internal wallet is not a hot wallet.

Reply:
Np